THIRD_PARTY_NOTICES.md

```diff
@@ -1317,6 +1321,215 @@ IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 
 ```
 
+## optionalDependencies
```

Comment:
This is manually added for now, so we'll want to make sure doesn't get deleted by the tool run in the future. Similar to the footer.

As such, when updating licenses I recommend selectively choosing lines to stage and tossing away errant deletes.